http_client.go

```diff
@@ -318,6 +319,14 @@ func (c *HTTPClient) Send(data []byte) (response []byte, err error) {
 		}
 	}
 
+	if (c.config.CloseConnectionOnResponseStatus > 0) {
```

Comment:
I prefer not to add additional options unless it is really needed. If there will be more requests like this, I will think about it.